tests/Avalonia.Build.Tasks.UnitTest/GenerateAvaloniaResourcesTaskTest.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.Build.Utilities;
+using Xunit;
+
+namespace Avalonia.Build.Tasks.UnitTest;
+
+public class GenerateAvaloniaResourcesTaskTest
+{
+    [Fact]
+    public void Does_Support_LinkBase_In_Avalonia_Resources_Generator()
+    {
+        var path = "path/to/resources";
+        var expected = "/" + path;
```

Comment:
The added test doesn't check the right thing: the expected path should be `/path/to/resources/root.xml`.